Assets/MixedRealityToolkit.SDK/Inspectors/UX/BoundingBox/BoundingBoxInspector.cs

```diff
@@ -0,0 +1,26 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+
+using Microsoft.MixedReality.Toolkit.UI;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    [CustomEditor(typeof(BoundingBox))]
+    [CanEditMultipleObjects]
+    public class BoundingBoxInspector : UnityEditor.Editor
```

Comment:
Nit: I wonder if we should make some abstract RenderHelpURLInspector base class for all these custom inspectors to avoid code dupe?

Reply:
yea I thought of that too. I wish I could just create one class and say all these objects will use that. But I still have to create an inspector for everyone and set [CustomEditor(typeof(Blah))]. 

I didn't go the base class route though because there is only two and inspectors have strong tendency to become specialized/custom during their lifetime.

Reply:
Yeah, I've been bitten by that same thing. I wish I could do [CustomEditor(typeof(Blah), typeof(Foo), typeof(Bar))]